client/state/initial-state.js

```diff
@@ -16,6 +17,8 @@ import config from 'config';
  */
 const debug = debugModule( 'calypso:state' );
 
+export const MAX_AGE = ms( '7 days' );
```

Comment:
I'm all for leveraging dependencies, but this seems a bit... overkill? :smile: Can't we just calculate this in advance and leave a comment `// 7 days` ?